packages/fxa-auth-server/lib/senders/emails/templates/verify/index.txt

```diff
@@ -1,8 +1,8 @@
-verify-title = "Activate the Firefox family of products"
+verify-title-2 = "Open the internet with Firefox"
 
-verify-description-plaintext = "Confirm your account and get the most out of Firefox everywhere you sign in."
+verify-description-plaintext = "Confirm your account and get the most out of Firefox everywhere you sign in starting with:"
```

Comment:
I think we need a new id here?

Reply:
Noting: ftl file would also need to be updated (both text and id).